tools/cloudformation-template/FlexeraAutomationPolicies.template

```diff
@@ -0,0 +1,416 @@
+  paramFlexeraZone:
+    Description: >-
+      The Flexera Zone which trust will be granted to.  The Organization ID should be located in this Flexera Zone.
+    Type: String
+    Default: app.flexera.com
+    AllowedValues:
+      - app.flexera.com
+      - app.flexera.eu
+      - app.flexeratest.com
+
+  # ParameterGroup: Parameters for the IAM Role that is created
+  paramRoleName:
+    Description: Name of the the IAM Role that will be created. If you plan to create more than one IAM Role (i.e. one for each Policy Template, or to trust multiple Orgs) you will need to modify this to prevent naming conflict.
+    Type: String
+    Default: FlexeraAutomationAccessRole
+    # IAM Role Name Max Length is 64chars
+    MaxLength: 64
+  paramRolePath:
+    Description: Path for the IAM Role that will be created. Generally does not need to be modified.
+    Type: String
+    Default: /
+  # TODO: Enable attaching existing permissions policy
+  # paramPermsExistingPermissionArn:
+  #   Description: 'Existing IAM Permission Policy ARN to attach to the IAM Role that will be created'
+  #   Type: String
+  #   AllowedPattern: "^arn:aws:iam::\d{12}:policy/.+"
+  #   ConstraintDescription: "Existing IAM Permission Policy ARN is optional and must match regex ^arn:aws:iam::\d{12}:policy/.+"
+
+  # ParameterGroup: Parameters to define Policy Template permissions on the IAM Role that is created
+  paramPermsAWSUnusedVolumes:
+    Description: 'What permissions should policies using "AWS Unused Volumes" Policy Template be granted on the IAM Role that will be created?'
+    Type: String
+    Default: Read Only
+    AllowedValues:
+      - No Access
+      - Read Only
+      - Read and Take Action
+  paramPermsAWSUnusedRDS:
+    Description: 'What permissions should policies using "AWS Unused RDS" Policy Template be granted on the IAM Role that will be created?'
+    Type: String
+    Default: Read Only
+    AllowedValues:
+      - No Access
+      - Read Only
+  paramPermsAWSUnusedIPAddresses:
+    Description: 'What permissions should policies using "AWS Unused IP Addresses" Policy Template be granted on the IAM Role that will be created?'
+    Type: String
+    Default: Read Only
+    AllowedValues:
+      - No Access
+      - Read Only
+      - Read and Take Action
+  paramPermsAWSOldSnapshots:
+    Description: 'What permissions should policies using "AWS Old Snapshots" Policy Template be granted on the IAM Role that will be created?'
+    Type: String
+    Default: Read Only
+    AllowedValues:
+      - No Access
+      - Read Only
+      - Read and Take Action
+  paramPermsAWSIdleComputeInstances:
+    Description: 'What permissions should policies using "AWS Idle Compute Instances" Policy Template be granted on the IAM Role that will be created?'
+    Type: String
+    Default: Read Only
+    AllowedValues:
+      - No Access
+      - Read Only
+
+Conditions:
+  # CreateExistingPolicyAttachment: !Not
```

Comment:
Are we leaving in these commented out sections?

Reply:
Fixed!  We can add that functionality in a new release but removed the commented blocks for now